### ISSUE TITLE
Converting CHECK outside loop to IF NOT ... RETURN considering BREAK-POINT

### ISSUE THREAD

**Issue by blackfish5** [SAP/abap-cleaner#135]
Could BREAK-POINT statement be considered the same way as local declarations at the beginning of the method so that CHECK in the following code is NOT converted to IF ... RETURN ... ENDIF?

```
METHOD process.
  DATA a TYPE c LENGTH 1.
  BREAK-POINT ID fins_rev_rec_acdocp.
  CHECK a IS INITIAL.
ENDMETHOD.
```

**Comment by jmgrassau** [comment]
Hi blackfish5, 

that's a good point, thanks for bringing this up! And I think this is valid for all [Checkpoints](https://help.sap.com/doc/abapdocu_latest_index_htm/latest/en-US/abencheckpoints.htm) (ASSERT, BREAK-POINT, LOG-POINT). However, rather than considering them like local declarations, I think it would be better to add an option like "[X] Keep CHECK statement independent of previous ASSERT, BREAK-POINT, or LOG-POINT". 

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi blackfish5, 

the new option is now available with version 1.7.0, which was just released! My initial wording was a bit complicated, hope this works better:

![image](https://github.com/SAP/abap-cleaner/assets/79639298/06fe7dd1-0ee6-423e-9a13-ec10f4787855)

Kind regards,
Jörg-Michael

**Comment by blackfish5** [comment]
I have updated to 1.7.1 and it works as expected. Thanks a lot for swift implementation.

BR
Michal
